crates/compiler/gen_wasm/src/backend.rs

```diff
@@ -1870,7 +1886,7 @@ impl<'a> WasmBackend<'a> {
     /// If the data size is known at compile time, pass it in comptime_data_size.
     /// If size is only known at runtime, push *data* size to the VM stack first.
     /// Leaves the *data* address on the VM stack
-    fn allocate_with_refcount(
+    pub fn allocate_with_refcount(
```

Comment:
Let's remove this `pub` since it ended up not being used. I'd like to know nobody is calling in to this.